src/node_library_utils.cpp

```diff
@@ -121,7 +121,11 @@ Status createTensorInfoMap(struct CustomNodeTensorInfo* info, int infoCount, std
 
         std::string name = std::string(info[i].name);
         auto precision = toInferenceEnginePrecision(info[i].precision);
-        shape_t shape(info[i].dims, info[i].dims + info[i].dimsCount);
+        ovms::Shape shape;
+        for (int j = 0; j < info[i].dimsCount; ++j) {
+            auto dim = info[i].dims[j];
+            shape.add(dim ? dim : DYNAMIC_DIMENSION);
```

Comment:
This could be encapsulated into `Shape::fromCustomNodeShape(std::uint64_t* arr, std::uint64_t size)`

Reply:
This way we can hide `DYNAMIC_DIMENSION` and the 0 ~ -1 ~ 0 logic